tests/api/stats_cfg/test_stats_cfg_rest.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2023 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests REST API statistics configuration."""
+
+import json
+from copy import deepcopy
+
+from flask import url_for
+from invenio_accounts.testutils import login_user_via_session
+from utils import postdata
+
+
+def test_stats_cfg_post_put_delete(client,
```

Comment:
all tests into this function return 403 --> so it's seems permissions tests that are already done into the specific file. You can remove this test.